tests/sentry/api/test_event_search.py

```diff
@@ -1107,21 +1126,24 @@ def test_negated_wildcard(self):
                 [["isNull", ["release"]], "=", 1],
                 [["match", ["release", "'(?i)^3\\.1\\..*$'"]], "!=", 1],
             ],
-            [["match", ["user.email", "'(?i)^.*\\@example\\.com$'"]], "=", 1],
+            [["match", ["user.email", "'(?i)^.*@example\\.com$'"]], "=", 1],
```

Comment:
Why do we stop escaping the `@` here? Is it just unnecessary?

Reply:
`@` Doesn't do anything in regex unless it is used as a delimiter.

Reply:
For extra peace of mind I've added a test that includes an `@`, and can confirm that `@` isn't supported by the regex library: [docs here](https://github.com/google/re2/wiki/Syntax)